libs/couch2pg/importer.js

```diff
@@ -0,0 +1,105 @@
+var _ = require('underscore'),
+    Promise = require('rsvp').Promise,
+    format = require('pg-format');
+
+var deleteDocuments = function(db, docIdsToDelete) {
+  if (docIdsToDelete && docIdsToDelete.length > 0) {
+    return db.query(
+      format('DELETE FROM couchdb WHERE doc->>\'_id\' in (%L)',
+        docIdsToDelete));
+  } else {
+    return Promise.resolve();
+  }
+};
+
+var storeSeq = function(db, seq) {
+  return db.query(format('UPDATE couchdb_progress SET seq = %L', seq));
```

Comment:
There is definitely a future where you'd store a JSON object for configuration / meta information, which would occur if we started storing more data. For example, I've considered storing the couchdb and pg db you connect to and having them added in a setup stage (which is different from the run stage), because right now I don't like that the sequence is stored away from and incidentally to the couchdb and pg urls, even though they are indelibly connected. I will probably raise a ticket for us to move to that at some point, but I didn't want to dig too deep right now.

However, I wouldn't store them in the couchdb table. I'd want it to be clear that the couchdb table is for data that is pulled from couchdb, and if you see a piece of data in that table, you know that it is currently (relative to the last sync) in couchdb in that exact state.